packages/mysql/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@pristine-ts/mysql",
+  "version": "0.0.335",
+  "description": "",
+  "module": "dist/lib/esm/mysql.module.js",
+  "main": "dist/lib/cjs/mysql.module.js",
+  "types": "dist/types/mysql.module.d.ts",
+  "scripts": {
+    "build": "tsc -p tsconfig.json && tsc -p tsconfig.cjs.json",
+    "prepublish": "npm run build",
+    "test": "jest",
+    "test:cov": "jest --coverage"
+  },
+  "files": [
+    "dist"
+  ],
+  "author": "",
+  "license": "ISC",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@pristine-ts/common": "file:../common",
+    "@pristine-ts/core": "file:../core",
+    "@pristine-ts/data-mapping-common": "file:../data-mapping-common",
+    "@pristine-ts/data-mapping": "file:../data-mapping",
+    "@pristine-ts/mysql-common": "file:../mysql-common",
+    "@pristine-ts/metadata": "^1.0.6",
+    "@pristine-ts/security": "file:../security",
+    "@pristine-ts/telemetry": "file:../telemetry",
+    "mysql2": "^3.10.0",
```

Comment:
I think even mysql2 is not used in this module ?

Reply:
Yes it is, that's how we make queries to the sql database.

Reply:
Ohh I miss read I thought this was the common-mysql package.